### PR TITLE
first commit sidebar

### PR DESCRIPTION
**Pull request by Kavitha0691**

### CODE REVIEW

File: js/shared.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        animalHeadings.forEach(function (heading) {
+            heading.classList.remove('active'); 
+        });
+    });
+});
+
+document.addEventListener("DOMContentLoaded", function() {
+    const sidebar = document.querySelector('.sidebar'); 
+
+    // sidebar.style.backgroundImage = "url('./images/zooimage.png')";
```
Comment (Svpriyaa2808) [inline js/shared.js:125]: you can remove these lines if not needed

File: styles/shared.css [hunk 2]
```diff
@@ -5,5 +5,287 @@
...
+
+.animal-description h3 {
+    display: flex;
+    width: 300px;
+    font-size: 25px;
+    margin-right: 20px;
+    color: #333;
+}
+
+
```
Comment (Svpriyaa2808) [inline styles/shared.css:145]: remove whitespaces

Comment (Vineethaveerapu) [reply styles/shared.css:145]: this is fixed now

**Review by Vineethaveerapu — APPROVED** [review]
nice pictures

File: images/cockatoo.png
Comment (Vineethaveerapu) [inline images/cockatoo.png:1]: can you find new image for this because it has stock image text

File: images/tasdevil.png
Comment (Vineethaveerapu) [inline images/tasdevil.png:1]: Could you also find an image for this without any text?

**Review by Vineethaveerapu — CHANGES_REQUESTED** [review]
can you update the images that i commented on.

File: images/cockatoo.png
Comment (Kavitha0691) [reply images/cockatoo.png:1]: Yes, sure I will try to find better image

File: images/tasdevil.png
Comment (Kavitha0691) [reply images/tasdevil.png:1]: I will do that

File: images/cockatoo.png
Comment (Kavitha0691) [reply images/cockatoo.png:1]: i updated the png icon which you mentioned.

File: js/shared.js (hunk 1 above)
Comment (Kavitha0691) [reply js/shared.js]: i removed that.

**Review by Vineethaveerapu — APPROVED** [review]

**Comment by Vineethaveerapu** [comment]
i checked and reviewed the changes so i am merging the branch

**Pull request merged**